### PR TITLE
LPS-156187 Test Automation Creation | master

### PR DESCRIPTION
**Pull request by ozysouza**
https://issues.liferay.com/browse/LPS-156187

cc @david-gutierrez-mesa 
cc @Tim-Cao

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: portal-web/test/functional/com/liferay/portalweb/macros/layout/pageeditor/PageEditor.macro [hunk 1]
```diff
@@ -4865,6 +4865,14 @@ return	Array.from(getWebElement("(//div[@aria-label='${fragmentName}'])[${index}
 			value1 = "${numberOfItems}");
 	}
 
+	macro viewSelectFromList {
+		Click(
+			key_text = "caret-double",
+			locator1 = "Icon#ANY");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/layout/pageeditor/PageEditor.macro:4871]: Add a new locator call Fragment#INPUT_TOGGLE
```
xpath=(//div[contains(@id,'fragment-')]/div[contains(@class,'${type}')]//button[*[name()='svg'][contains(@class,'icon-caret-double')]])[${index}]
```

File: portal-web/test/functional/com/liferay/portalweb/macros/layout/pageeditor/PageEditor.macro [hunk 2]
```diff
@@ -4865,6 +4865,14 @@ return	Array.from(getWebElement("(//div[@aria-label='${fragmentName}'])[${index}
 			value1 = "${numberOfItems}");
 	}
 
+	macro viewSelectFromList {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/layout/pageeditor/PageEditor.macro:4868]: No need to group in a macro since it is rare case

**Review by Tim-Cao — COMMENTED** [review]
Hi @ozysouza, would you mind updating according to comments? Thanks

File: portal-web/test/functional/com/liferay/portalweb/macros/layout/pageeditor/PageEditor.macro (hunk 2 above)
Comment (ozysouza) [reply portal-web/test/functional/com/liferay/portalweb/macros/layout/pageeditor/PageEditor.macro:4868]: Ok. I will remove the macro

File: portal-web/test/functional/com/liferay/portalweb/paths/portlet/fragments/Fragment.path [hunk 3]
```diff
@@ -147,7 +147,7 @@
 </tr>
 <tr>
 	<td>INPUT_LABEL</td>
-	<td>xpath=(//div[contains(@id,'fragment-')]//div[contains(@class,'form')]//label[contains(@for,'${type}')][normalize-space()='${fieldLabel}'])[${index}]</td>
+	<td>xpath=(//div[contains(@id,'fragment-')]//div[contains(@class,'form')]//label[contains(@for,'${type}')][normalize-space()='${fieldLabel}'])[${index}] | (//div[contains(@id,'fragment-')]//div[contains(@class,'form')]//label[normalize-space()='${fieldLabel}'])[${index}]</td>
```
[the remark this answers is not in the document]
Comment (ozysouza) [reply portal-web/test/functional/com/liferay/portalweb/paths/portlet/fragments/Fragment.path:150]: Okay, Thanks for the heads up!

**Comment by ozysouza** [comment]
> Hi @ozysouza, would you mind updating according to comments? Thanks

Hi @Tim-Cao, PR updated. I've also changed other paths to work with select from list. Would you have a look at? Thanks!

**Comment by Tim-Cao** [comment]
See changes @ozysouza
https://github.com/liferay-echo/liferay-portal/pull/8740

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**